lib/Jsrt/ChakraCore.h

```diff
@@ -104,6 +104,53 @@ typedef JsErrorCode(CHAKRA_CALLBACK * FetchImportedModuleFromScriptCallBack)(_In
 /// </returns>
 typedef JsErrorCode(CHAKRA_CALLBACK * NotifyModuleReadyCallback)(_In_opt_ JsModuleRecord referencingModule, _In_opt_ JsValueRef exceptionVar);
 
+/// <summary>
+///     A structure containing information about a native function callback.
+/// </summary>
+struct JsNativeFunctionInfo
```

Comment:
Is this struct one that we will be able to add fields to in the future, even if this API becomes part of Windows? (Along the same lines as my other comment about needing future updates to this API surface).

Reply:
We should be able to add more fields to the struct later if we want to. If it becomes part of Windows API surface, changing it will become very difficult. We'll have to go through Windows API review board.